python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/platform.schema.yml

```diff
@@ -41,6 +41,52 @@ keys:
                   keys:
                     name:
                       type: str
+                    ingress:
+                      type: dict
+                      keys:
+                        priority_groups:
+                          type: list
+                          items:
+                            type: dict
+                            keys:
+                              id:
+                                type: int
+                                convert_types:
+                                  - str
+                                description: Priority-group group number.
+                                required: true
```

Comment:
make `id` the primary_key instead of using required

Reply:
Done!